<launch>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find unitree_lidar_ros)/rviz/view.rviz" />

  <node pkg="unitree_lidar_ros" type="unitree_lidar_ros_node"  name="unitree_lidar_ros_node" respawn="true" output="screen">
    <!-- CRITICAL: This node reads parameters using absolute names like
         /unitree_lidar_ros_node/publish_tf (not private params). So we must
         load YAML into the *global* /unitree_lidar_ros_node namespace. -->
    <rosparam command="load" ns="/unitree_lidar_ros_node" file="$(find unitree_lidar_ros)/config/config.yaml"/>
  </node>

</launch>

